packages/client/src/pages/logout/ui.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import {
+  Button,
+  Divider,
+  Form,
+  message,
+  Modal,
+  Popconfirm,
+  Typography,
+} from 'antd'
+import { Link, useNavigate } from 'react-router-dom'
+import { routesPath } from 'processes/routes'
+import { loginFx, logoutFx } from 'processes/http/httpService'
+
+export const Logout: React.FC = () => {
+  const [messageApi, contextHolder] = message.useMessage()
+  const navigate = useNavigate()
+
+  const handleCancel = () => {
+    navigate(routesPath.home)
+  }
+
+  const onFinish = async () => {
+    try {
+      await logoutFx()
+      navigate(routesPath.home)
+    } catch (e) {
+      messageApi.open({
+        type: 'error',
+        content: (e as { response: { data: { reason: string } } }).response.data
+          .reason,
+      })
+    }
+  }
+
+  return (
+    <>
+      {contextHolder}
+      <Modal title="Выход" open={true} onCancel={handleCancel} footer={null}>
```

Comment:
Подсказка:
В пропсах реакта если булевое значение `true`, то можно его не указывать
```suggestion
      <Modal title="Выход" open onCancel={handleCancel} footer={null}>
```

Reply:
Принято